docs: add Experimentation guides (#7993)
Flagsmith/flagsmith: docs/docs/experimentation/connect-a-warehouse.md

## Connect the Flagsmith warehouse

The **Flagsmith** warehouse is managed and hosted by Flagsmith. There is nothing to provision or configure.

1. Go to **Environment Settings > Warehouse**.
2. Select **Flagsmith** and click **Enable**, then confirm.

![The Warehouse tab with the Flagsmith managed warehouse selected](/img/experimentation/warehouse-tab.png)

The connection is created immediately. To verify that events can flow, click **Send your first event** and Flagsmith
sends a test event on your behalf. The connection shows **Pending Connection** while it waits for the first event, and
switches to **Connected** as soon as the environment's first event arrives; an event sent from your own application
counts too, not just the test event. Processing the first event can take up to a few hours, and the status refreshes
about once a minute while you keep the Warehouse tab open.

Once connected, the warehouse card shows the total number of events received and the number of unique event names, which
is useful for checking that your instrumentation is arriving.

<!-- Screenshot: connected warehouse card with event stats -->

> ### Connection statuses

Reviewer: Maybe we should tell the users where to look for these, it took me minute to find it

Author: Addressed in [f29e156](https://github.com/Flagsmith/flagsmith/pull/7993/commits/f29e156c18e0213a932d81c781009b408c089559)

---
title: Connect a Warehouse
sidebar_label: Connect a Warehouse
sidebar_position: 2
description: Enable the managed Flagsmith warehouse to store and query experiment events.
---

:::info Enterprise beta

Experimentation is in beta on **Enterprise** plans. [Get in touch](https://www.flagsmith.com/contact-us) to join.

:::

Experiment data (exposures and conversion events) is stored and queried in a data warehouse. Each environment has one
warehouse connection, and it must be in place before you can create experiments.

## Connect the Flagsmith warehouse

The **Flagsmith** warehouse is managed and hosted by Flagsmith. There is nothing to provision or configure.

1. Go to **Environment Settings > Warehouse**.

> 2. Select **Flagsmith** and click **Enable**, then confirm.

Reviewer 2: As an end user I would want to be reassured that there is no hidden cost involved here.

![The Warehouse tab with the Flagsmith managed warehouse selected](/img/experimentation/warehouse-tab.png)

The connection is created immediately. To verify that events can flow, click **Send your first event** and Flagsmith
sends a test event on your behalf. The connection shows **Pending Connection** while it waits for the first event, and
switches to **Connected** as soon as the environment's first event arrives; an event sent from your own application

> counts too, not just the test event. Processing the first event can take up to a few hours, and the status refreshes

Reviewer 2: Might be worth elevating this to a :::Note::: (Processing the first event can take up to a few hours)

about once a minute while you keep the Warehouse tab open.

Once connected, the warehouse card shows the total number of events received and the number of unique event names, which

> is useful for checking that your instrumentation is arriving.

Reviewer 2: ```suggestion
is useful to verify that instrumentation data is arriving.
```

<!-- Screenshot: connected warehouse card with event stats -->

### Connection statuses

The current status is shown on the warehouse card in **Environment Settings > Warehouse**.

- **Created**: the connection exists but no events have been received yet.
- **Pending Connection**: waiting for the first event to arrive.
- **Connected**: events have been received; you are ready to run experiments.

> - **Errored**: something went wrong; contact support.

Reviewer 2: Add link to support page /support/
